ambry-store/src/main/java/com/github/ambry/store/HybridCompactionPolicy.java

```diff
@@ -80,16 +84,23 @@ public CompactionDetails getCompactionDetails(long totalCapacity, long usedCapac
    * @param storeId id of the BlobStore
    * @return {@link CompactionPolicySwitchInfo} gets from map or recover from file if needed.
    */
-  private CompactionPolicySwitchInfo getCompactionPolicySwitchInfo(String storeId, String dataDir, BlobStoreStats blobStoreStats) {
+  private CompactionPolicySwitchInfo getCompactionPolicySwitchInfo(String storeId, String dataDir,
+      BlobStoreStats blobStoreStats) {
     if (!blobToCompactionPolicySwitchInfoMap.containsKey(storeId)) {
-      File file = new File(Paths.get(dataDir, COMPACT_POLICY_INFO_PATH).toString());
+      File file = new File(Paths.get(dataDir, COMPACT_POLICY_INFO_PATH_V2).toString());
+      File oldFile = new File(Paths.get(dataDir, COMPACT_POLICY_INFO_PATH).toString());
```

Comment:
You can create File without Paths:
```
new File(dataDir, COMPACT_POLICY_INFO_PATH_V2)
```

Also,
```
  private static final String COMPACT_POLICY_INFO_PATH_V2 = File.separator + "compactionPolicyInfoV2.json";
```
I feel `File.separator` is not necessary.

Reply:
When we print dataDir, it won't have the slash at the end right? So without the File.separator, it will be like "/mnt/u001/ambrydata/230compactionPolicyInfo.json"?

Reply:
Updated.